rest/blip_api_test.go

```diff
@@ -4532,3 +4532,109 @@ func TestBlipRevokeNonExistentRole(t *testing.T) {
 		return rt.GetDatabase().DbStats.CBLReplicationPull().NumPullReplCaughtUp.Value()
 	}, 1)
 }
+
+func TestBlipGetCollections(t *testing.T) {
+
+	base.SetUpTestLogging(t, base.LevelDebug, base.KeyAll)
+
+	rt := NewRestTester(t, &RestTesterConfig{
+		guestEnabled: true,
+		DatabaseConfig: &DatabaseConfig{
+			DbConfig: DbConfig{
+				Scopes: ScopesConfig{
+					"fooScope": ScopeConfig{
+						Collections: map[string]CollectionConfig{
+							"fooCollection": {},
+						},
+					},
+				},
+			},
+		},
+	})
+	defer rt.Close()
+
+	btc, err := NewBlipTesterClientOptsWithRT(t, rt, nil)
+	require.NoError(t, err)
+	defer btc.Close()
+
+	checkpointBody := db.Body{}
+	checkpointID1 := "checkpoint1"
+	dbInstance := db.Database{DatabaseContext: rt.GetDatabase()}
+	//TODO, use db.checkpointDocIDPrefix exported ?
+	revID, err := dbInstance.PutSpecial(db.DocTypeLocal, "checkpoint/"+checkpointID1, checkpointBody)
```

Comment:
Agreed - although we should maybe move the constant out of the ISGR-specific (ActiveReplicator) file and into maybe `blip_sync_messages.go`